boards/arduino-atmega-common/Makefile.features

```diff
@@ -4,6 +4,7 @@ FEATURES_PROVIDED += periph_i2c
 FEATURES_PROVIDED += periph_spi
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_pm
```

Comment:
should be moved to the (possible shared) CPU features file (as implemented for all atmega CPUs, right?)

Reply:
done